contracts/strategy/liquity/LiquityStrategy.sol

```diff
@@ -364,13 +366,39 @@ contract LiquityStrategy is
         external
         override(IStrategy)
         onlyManager
-        returns (bool)
+        returns (uint256)
+    {
+        uint256 ethBalance = address(this).balance;
+
+        if (ethBalance == 0) return 0;
+
+        uint256 amountInETH = _getExchangeAmountInEth(_amount);
+
+        uint256 ethToTransfer = amountInETH > ethBalance
+            ? ethBalance
+            : amountInETH;
+
+        _sendEth(_to, ethToTransfer);
+
+        uint256 equivalentAmountInUnderlying = amountInETH > ethBalance
```

Comment:
I think you don't need the condition because it would end up as 100%, right?

Reply:
you are right, it's a repeating condition anyways, I'll refactor it out